concent-builder/configure.yml

```diff
@@ -76,8 +77,8 @@
         - name:    Install dependencies from testing
```

Comment:
Please hard-code versions of packages that come from PPAs (and not official repositories): docker, kubectl, Google Cloud SDK

Reply:
Done.